File: Core/Entities/Photo.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    [Table("Photos")]
    public class Photo
    {
        public Guid Id { get; set; }
        public string Url { get; set; } = null!;
        public bool IsMain { get; set; }
        public bool IsCover { get; set; }
        public string? PublicId { get; set; }
        public Guid? AppUserId { get; set; }
        public AppUser? AppUser { get; set; }
        public Guid? ReviewPostId { get; set; }
        public ReviewPost? ReviewPost { get; set; }
    }
}

File: client/src/app/shared/models/photo.ts
import { ReviewPostModel } from './review/review-post';
import { User } from './user';

export interface Photo {
  id: string;
  url: string;
  isMain: boolean;
  isCover: boolean;
  publicId: string | null;
  appUserId: string | null;
  appUser: User | null;
  reviewPostId: string | null;
  reviewPost: ReviewPostModel | null;
}
